share/jupyter/voila/templates/default/nbconvert_templates/lab.tpl

```diff
@@ -1,4 +1,4 @@
-{%- extends 'display_priority.tpl' -%}
```

Comment:
With https://github.com/jupyter/nbconvert/pull/1173 we could again use the tpl, if we care about backwards compatibility

Reply:
I reverted to `.tpl`.